lego-webapp/components/ShareButton/index.tsx

```
@@ -0,0 +1,39 @@
+  url: string;
+};
+
+const ShareButton: React.FC<Props> = ({ title, url }) => {
+  const dispatch = useAppDispatch();
+  const handleShare = async () => {
+    if (navigator?.canShare?.({ title, url })) {
+      navigator.share({ title, url });
+    } else {
+      {
+        navigator.clipboard.writeText(url);
+      }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)